### DISCUSSION TITLE
Can't create second project on the Hobby Plan

### DISCUSSION THREAD

**Discussion by awmartin** [vercel/community#7246] [Help] [answered]
### Summary

I'm evaluating Vercel, and I have one project deployed from GitHub on Vercel already on a Hobby Plan.

However, when I attempt to create a second project for my single existing team, Vercel prompts me to create a new team and upgrade to the Pro plan, even though the [Hobby Plan description](https://vercel.com/docs/accounts/plans/hobby) claims I can create 200 projects.

What am I missing?

### Steps to Reproduce

1. Create a new account with a Hobby Plan.
2. Create a team and project.
3. Attempt to create a new project for that team.
4. Select a GitHub repo.
5. I see a page that asserts I should create a new team and upgrade to the Pro plan.

**Comment by amyegan** [comment, marked as answer]
Hey @awmartin. I responded to your other comments, but it'll be easiest if we keep the conversation in this dedicated discussion. 😄 

You can currently have up to 200 projects on Hobby plans. I think you might have run into a limit.

Vercel does not allow Hobby projects connected to Git repos owned by a Git organization (e.g. GitHub Organization, GitLab group, Bitbucket team). That is a common reason that people see a prompt instructing them to upgrade to a Pro plan.

But it's possible that something else is preventing you from creating the new project. Can you walk me through what you tried and what you see on the screen? That will help me understand exactly what is going wrong in your case.

**Reply by awmartin** [reply]
The restriction regarding GitHub orgs is likely the problem.

The confusing part is that I already have a project deployed with code hosted by a GitHub org on the Hobby Plan, so I didn't have that expectation at all.

Again, I think the restriction is rather silly, but I'd recommend posting that constraint directly in the Hobby Plan documentation somewhere (https://vercel.com/docs/accounts/plans/hobby). And best in the UI itself to help us understand why this behavior is so.

**Reply by amyegan** [reply]
Can you walk me through the steps you tried? It's possible you uncovered a bug and I want to be sure I'm investigating the right place.

**Reply by amyegan** [reply]
It would also be helpful to know the exact message you see

**Reply by awmartin** [reply]
Sure, but there's really not much to tell.

1. Navigate to my team page: https://vercel.com/spatialpixel
2. Observe the page shows a successfully deployed project from my GitHub org.
3. Click on "Add New…" in the upper right.
4. Click on "Project" from the resulting dropdown.
5. Click on "Import" next to the GitHub repo I'd like to deploy. This is a repo in the same GitHub org as my currently deployed project.
6. Observe a page that says "You're almost done." It says "Create a Team" and "Configure Project." There is no error or warning message.

![vercel-new-team](https://github.com/vercel/community/assets/57281/4df6f95d-f8cf-490a-a4e3-403256498fc2)

Again, I think the confusing part for me was it wasn't clear from the UI that there was a constraint, in addition to the fact I was somehow able to deploy a project from this GitHub org already.

I guess I would have expected a message that said something like, "Oops! You're trying to deploy from a GitHub org, which isn't allowed on the Hobby Plan. Here are some options... Fork the repo to your personal account. Upgrade to the Pro Plan. … Etc."

If the bug is "fixed," meaning my current project is taken down, I think that means I won't be using Vercel just yet. In the time I've taken to learn about this, I could have deployed a dozen projects to Netlify.

But now that I know what the problem likely is, I feel like it's a pretty 101-level issue that everyone seems to know, so sorry to bother you.

**Reply by awmartin** [reply]
Hmm, I think I know what happened. Perhaps I added the original project under Pro trial?

**Reply by amyegan** [reply]
If you recently had a Pro trial, that's a plausible explanation

[a comment by an automated account is not shown]
